src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Delete.cshtml.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using APIViewWeb.Models;
+using Azure.Storage.Blobs;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.Extensions.Configuration;
+
+namespace APIViewWeb.Pages.Assemblies
+{
+    public class DeleteModel : PageModel
+    {
+        public DeleteModel(IConfiguration configuration)
+        {
+            Configuration = configuration;
+            string connectionString = Configuration.GetValue<String>("APIVIEW_STORAGE");
```

Comment:
Extract all Azure Storage logic into a separate type that would have logic to store/load models from the blob storage.